dart-github-dart-lang/.travis.yml

```diff
@@ -1,14 +1,24 @@
 language: dart
 
-script: ./tool/travis.sh
-
 # Speed up builds by using containerization. Disable this if you need to use
 # sudo in your scripts.
 sudo: false
 
+dart:
+ - dev
+ - stable
+
+dart_task:
+ - test: --platform dartium
```

Comment:
I'm not familiar w/ these configurations options. Does this mean that we run through each task serially?

We may want to add a few short lines of documentation in order to familiarize developers with what these do:

```
# Validate that the code is formatted.
- dartfmt
```

Also, perhaps leave the dartium and firefox tasks commented out? They'll still be good templates for people in terms of how to enable support, without having to run for each project.